bot-twitter/src/keeper.ts

```diff
@@ -0,0 +1,101 @@
+import { AuctionInitialInfo } from 'auctions-core/src/types';
+import getSigner, { createSigner, setSigner } from 'auctions-core/src/signer';
+import { bidOnTheAuction, enrichAuction } from 'auctions-core/src/auctions';
+import {
+    authorizeCollateral,
+    authorizeWallet,
+    getCollateralAuthorizationStatus,
+    getWalletAuthorizationStatus,
+} from 'auctions-core/src/authorizations';
+import { ETHEREUM_NETWORK, KEEPER_MIN_PROFIT_DAI, KEEPER_WALLET_PRIVATE_KEY } from './variables';
+import { setupWallet } from '~/src/authorizations';
+
+function areKeeperVariablesSet(): boolean {
+    if (!KEEPER_WALLET_PRIVATE_KEY) {
+        return false;
+    }
+    return Number.isNaN(KEEPER_MIN_PROFIT_DAI);
+}
+
+export async function setupKeeper() {
+    if (!areKeeperVariablesSet()) {
+        console.warn('keeper: keeper variables are not set. Please check configuration.');
+        return;
+    }
```

Comment:
I've proved 2 options: throwing errors and a global variable, but you implemented something else. Can you elaborate why?

Reply:
I made the function throw an error.